### PR TITLE
Query Summary: Demographics Pies Card

### PR DESCRIPTION
**Pull request by nyanofthemoon**

### CODE REVIEW

**Comment by ciaranschutte** [comment]
Can we move it to here: https://github.com/kids-first/kf-portal-ui/tree/next/src/chartkit/components?

File: stories/ChartKit/index.js [hunk 1]
```diff
@@ -1,2 +1,5 @@
+// @TODO File import issues Commented ou for now
 //export { default } from './HorizontalBar';
-export { default } from './ProgressBar';
+
+// @TODO  File import issues due to analytics code bundled into the ProgressBar component
+//export { default } from './ProgressBar';
```
Comment (hlminh2000) [inline stories/ChartKit/index.js]: we can put this back now :) just drop https://github.com/kids-first/kf-portal-ui/pull/1129/commits/8d5eccb205c2f3f3f2f95f49d0acaa00781f60f7 or revert it 📦

Comment (nyanofthemoon) [reply stories/ChartKit/index.js]: merged to latest

**Comment by hlminh2000** [comment]
Only thought is I think we can make the colours go down the scale (#gradientDescent #MLjokes #sorryIhaveToBeAMillennial) more smoothly as in the design. @joneubank worked on the chart in the user dashboard so he should have some insights on the implementation.
screen shot 2019-01-29 at 1 43 18 pm
Design:
screen shot 2019-01-29 at 1 53 38 pm

There's some layout issues but it's probably easier for @ciaranschutte to address together with the rest of the page layout.

otherwise lgtm 👍 any thoughts @ciaranschutte

**Comment by ciaranschutte** [comment]
> Only thought is I think we can make the colours go down the scale (**Gradient Descent!!!** #MLjokes #sorryIhaveToBeAMillennial) more smoothly as in the design. @joneubank worked on the chart in the 
> otherwise lgtm  any thoughts @ciaranschutte ?

Yup looks good! Agree with gradient comment. Even though the pun was overfitting ;D
Layout I can take a look at.

**Comment by nyanofthemoon** [comment]
@hlminh2000 I re-ordered the mock data, it depends on the order in the array... :)

**Comment by nyanofthemoon** [comment]
> > Only thought is I think we can make the colours go down the scale (**Gradient Descent!!!** #MLjokes #sorryIhaveToBeAMillennial) more smoothly as in the design. @joneubank worked on the chart in the
> > otherwise lgtm  any thoughts @ciaranschutte ?
> 
> Yup looks good! Agree with gradient comment. Even though the pun was overfitting ;D
> Layout I can take a look at.

It's ok. Karma always comes back to bite you one day.
